sdk/ai/ai-inference-rest/review/ai-inference.api.md

```diff
@@ -193,9 +193,44 @@ export interface CompletionsUsageOutput {
 }
 
 // @public
-function createClient(endpointParam: string, credentials: TokenCredential | KeyCredential, options?: ClientOptions): ModelClient;
+function createClient(endpointParam: string, credentials: TokenCredential | KeyCredential, { apiVersion, ...options }?: ModelClientOptions): ModelClient;
 export default createClient;
 
+// @public
+export type EmbeddingEncodingFormat = string;
+
+// @public
+export interface EmbeddingInput {
+    image: string;
```

Comment:
is it image embedding only?

Reply:
Ok looks like it is for image embedding only, could we make the model name more indicative?

Reply:
@dargilco Could we change this to "ImageEmbeddingInput" in the TS?

Reply:
@glharper Can you please check how this is called in the v2 OAI TypeSpec? (it may be defined in line, without a model name, but let's check)

Reply:
@dargilco [Here](https://github.com/Azure/azure-rest-api-specs/blob/feature/aoai-2024-07-01-preview-tsp/specification/cognitiveservices/OpenAI.Inference/routes/embeddings.tsp) is the route definition, that [Embeddings object](https://github.com/Azure/azure-rest-api-specs/blob/feature/aoai-2024-07-01-preview-tsp/specification/cognitiveservices/OpenAI.Inference/models/embeddings_create.tsp) only seems to refer to text embeddings in the [Input member](https://github.com/Azure/azure-rest-api-specs/blob/cb1d3d6e252c109c652cb939235d9fd91954d589/specification/cognitiveservices/OpenAI.Inference/models/embeddings_create.tsp#L40).